chanzuckerberg/idseq-web
# Add filter tags for taxon, thresholds and categories

# Description

Add filter tags for taxon, thresholds and categories

# Tests

* Set filters to add tags
* Remove filters to remove tags
* Verify that report data gets refreshed according to results
